hack/crds/hive.openshift.io_clusterimagesets.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: clusterimagesets.hive.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.releaseImage
+    name: Release
+    type: string
+  group: hive.openshift.io
+  names:
+    kind: ClusterImageSet
+    listKind: ClusterImageSetList
+    plural: clusterimagesets
+    shortNames:
+    - imgset
+    singular: clusterimageset
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: ClusterImageSet is the Schema for the clusterimagesets API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Could you use the pipe sign for [multi-lines](https://yaml-multiline.info/) instead of quotes?